quartz-js/components/select/select.controller.js

```diff
@@ -45,6 +45,10 @@ vhxm.components.shared.select.controller = function(opts) {
     });
 
     api(self);
+
+    $(window).on('quartz:select:open', () => {
```

Comment:
If there is a standard way to tear down components when they're removed, then a minor memory leak can be prevented by calling `.off(...` during teardown.

If not, I'd say this isn't a big deal since all the stray listeners disappear when the page changes.